recipes/cdms2/build.sh

```diff
@@ -0,0 +1,8 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC ${CFLAGS}"
+export CXXLAGS="${CFLAGS} ${CXXLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export UVCDAT_ANONYMOUS_LOG=False
+
+
+python setup.py install
```

Comment:
If this uses setuptools you can try `--single-version-externally-managed --record record.txt`. (No need to do it now, it can be tested in the feedstock later.)